vmdb/lib/token_manager.rb

```diff
@@ -28,7 +22,13 @@ def self.class_initialize(name = DEFAULT_NS, options = {})
   end
 
   def self.global_token_store(namespace)
-    @global_token_store[namespace] ||= {}
+    @token_caches[namespace] ||= begin
+      memcache_server = VMDB::Config.new("vmdb").config[:session][:memcache_server] || "127.0.0.1:11221"
+      Dalli::Client.new(memcache_server,
```

Comment:
This won't be available in test mode (i.e. Travis).  Do we have to do something similar to https://github.com/ManageIQ/manageiq/blob/master/vmdb/config/initializers/session_store.rb#L1-L2 ?